### PR TITLE
doc for pg-history-viewer

### PR DESCRIPTION
**Pull request by lbartoletti**
This PR adds a minimal documentation about the use of PG history viewer in QGEP

Same as https://github.com/qwat/docs/pull/8

cc @ponceta @m-kuhn

### CODE REVIEW

**Comment by m-kuhn** [comment]
Thanks @lbartoletti 

Almost all screenshots are from qwat currently.
@ponceta do you have this in production and could bless us with some screenshots with qgep tables/data?

**Comment by ponceta** [comment]
pg_hv_configuration
pv_hv_history
pv_hv_replay_after_bug
pv_hv_replay_before

Here are the screenshots from history viewer. Sadly, the replay function does not work here.

**Comment by ponceta** [comment]
We should maybe enhance it with how to start and stop logging features :

```
To start logging a table, here 'qgep_od.wastewater_structure':

qgep_sys.audit_table('qgep_od.wastewater_structure');

or for a view; here 'qgep_od.vw_qgep_reach':

qgep_sys.audit_view('qgep_od.vw_qgep_reach', 'true'::boolean, '{}'::text[], '{id}'::text[]);
```

from : https://github.com/qwat/pg-history-viewer readme

**Comment by ponceta** [comment]
On my db I started auditing with :

https://github.com/VilleDePully/qgep_sigip/blob/master/migration/audit.sql

**Comment by m-kuhn** [comment]
@lbartoletti what's your plan to finalize this?

File: en/user-guide/history/history.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- 1. Part identifying the tools used to filter modifications in the database.
+- 2. Logged events with the modification date, the table, the action type "Update/Delete/Insert", the application and the user who made the modification.
+- 3. The view comparing the data before and after the change. The red lines are the ones modified.
+- 4. If the geometry has been modified, a canvas will show the difference.
+
+Replay function
+---------------
+
+.. note::
+  Doesn't works yet for QGEP.
```
Comment (m-kuhn) [inline en/user-guide/history/history.rst]:
```suggestion
  Doesn't work yet for QGEP.
```

Is there an issue that describes what's missing?

Comment (lbartoletti) [reply en/user-guide/history/history.rst]: https://github.com/QGEP/docs/issues/112 Here :)

I'll work on it.

**Comment by ponceta** [comment]
@m-kuhn @lbartoletti thanks! A new tool for QGEP efficiency!

We are currently fixing some trouble with the replay function but this is already great.

**Pull request merged**